modules/ding_reservation/ding_reservation.module

```diff
@@ -614,15 +620,20 @@ function ding_reservation_reservations_form($form, &$form_state, $reservations,
         break;
     }
 
-    if (in_array($type, array(DING_RESERVATION_NOT_READY, DING_RESERVATION_INTERLIBRARY_LOANS))) {
-      // Set reservation expire message.
-      $expire = strtotime(check_plain($reservation->expiry));
-      if ($expire - variable_get('reservation_expire', 604800) <= time()) {
-        $item['#material_message'] = array(
-          'message' => t('This reservation is about to expire.'),
-          'class' => 'messages warning',
-        );
-        $item['#weight'] = -30;
+    if (empty($int_per)) {
+      if (in_array(
+        $type,
+        array(DING_RESERVATION_NOT_READY, DING_RESERVATION_INTERLIBRARY_LOANS)
+      )) {
```

Comment:
Maybe it will be more clear to move the in_array statement out of the if condition, eg.
$reservation_eligibility = in_array($type, array(DING_RESERVATION_NOT_READY, DING_RESERVATION_INTERLIBRARY_LOANS));
And make if statement like this: if ($reservation_elibitilty) {
 ...
}